elements/lisk-p2p/src/utils.ts

```diff
@@ -0,0 +1,287 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import { hash } from '@liskhq/lisk-cryptography';
+import { isIPv4 } from 'net';
+
+const SECRET_BUFFER_LENGTH = 4;
+const NETWORK_BUFFER_LENGTH = 1;
+const PREFIX_BUFFER_LENGTH = 1;
+const BYTES_4 = 4;
+const BYTES_16 = 16;
+const BYTES_64 = 64;
+const BYTES_128 = 128;
+const NEW_PEERS = 'new';
+const TRIED_PEERS = 'tried';
+
+export enum NETWORK {
+	NET_IPV4 = 0,
+	NET_PRIVATE,
+	NET_LOCAL,
+	NET_OTHER,
+}
```

Comment:
Can we make this a string enum for consistency with other parts of the lisk-p2p code?
The underlying reason is that integer enums are a pain to debug. The performance gain of using integers for this is insignificant when you consider the big picture.

Reply:
I would prefer this to remain an integer as the byte value is written into the buffer to be hashed later. In the bitcoin implementation they also use integers, and I would like to keep it consistent with that approach.